pd_front_end.py

```diff
@@ -102,16 +101,17 @@
 fsps_metals = np.loadtxt(cfg.par.metallicity_legend)
 N_METAL_BINS = len(fsps_metals)
 
-if par.FORCE_BINNING == False:
-    stellar_nu, stellar_fnu, disk_fnu, bulge_fnu = sg.allstars_sed_gen(stars_list, diskstars_list, bulgestars_list, ds.cosmological_simulation, sp)
```

Comment:
why did this line dissapear (and where is this allstars_sed_gen being called now?)

Reply:
It's being called inside direct_add_stars now.